public/video-ui/src/actions/WorkflowActions/getStatuses.js

```diff
@@ -0,0 +1,34 @@
+import WorkflowApi from '../../services/WorkflowApi';
+
+function requestStatuses() {
+  return {
+    type: 'WORKFLOW_STATUSES_GET_REQUEST',
+    receivedAt: Date.now()
+  };
+}
+
+function receiveStatuses(statuses) {
+  return {
+    type: 'WORKFLOW_STATUSES_GET_RECEIVE',
+    receivedAt: Date.now(),
+    statuses
+  };
+}
+
+function errorReceivingStatuses(error) {
+  return {
+    type: 'SHOW_ERROR',
+    message: `Could not get Workflow statuses. <a href="${WorkflowApi.workflowUrl}" target="_blank" rel="noopener">Open Workflow to get a cookie.</a>`,
```

Comment:
Should we check what the error is before displaying a message?

Reply:
Was copying https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/actions/WorkflowActions/getSections.js#L18-L25.

I've only seen this problem in DEV because we're talking to Workflow CODE - I'm not sure how happy panda is with that...